releases/v20.2.0.md

```diff
@@ -0,0 +1,139 @@
+{{site.data.alerts.end}}
+
+- [CockroachCloud](#cockroachcloud)
+- [SQL](#sql)
+- [Recovery and I/O](#recovery-and-io)
+- [Deployment and Operations](#deployment-and-operations)
+- [Observability](#observability)
+- [Backward-incompatible changes](#backward-incompatible-changes)
+- [Known limitations](#known-limitations)
+- [Education](#education)
+
+<style>
+    table td:first-child {
+        min-width: 100px !important;
+    }
+    table td:nth-child(2) {
+        min-width: 200px !important;
+    }
+</style>
+
+### CockroachCloud
+
+- Disaster Recovery of CockroachCloud Clusters
+
+### SQL
+
+Version | Feature | Description
+--------|---------|------------
+Core | **Spatial Support** | CockroachDB now supports [spatial data types](../v20.2/spatial-data.html), [spatial indexes](../v20.2/spatial-indexes.html), and [spatial functions](../v20.2/functions-and-operators#spatial-functions), as well as the ability to migrating spatial data from various formats such as [Shapefiles](../v20.2/migrate-from-shapefiles), [GeoJSON](../v20.2/migrate-from-geojson), [GeoPackages](../v20.2/migrate-from-geopackage), and [OpenStreetMap](../v20.2/migrate-from-openstreetmap).
+Core | **User-Defined Schemas** | You can now [create user-defined logical schemas](../v20.2/create-schema.html), as well [alter user-defined schemas](../v20.2/alter-schema.html), [drop user-defined schemas](../v20.2/drop-schema.html), and [convert databases to user-defined schemas](../v20.2/convert-to-schema.html).
+Core | **Partial Indexes** | You can now use [partial indexes](../v20.2/partial-indexes.html) to specify a subset of rows and columns in a table that evaluate to true on a `WHERE` filter defined at [index creation](#creation).
+Core | **`ENUM` data types** | CockroachDB now supports the creation and management of [user-defined `ENUM` data types](../v20.2/enum.html) consisting of sets of enumerated, static values.
+Core | **Materialized Views** | CockroachDB now supports [materialized views](../v20.2/views.html#materialized-views), or views that store their selection query results on-disk.
+Core | **View Replacement** | CockroachDB now supports [replacing an existing view](../v20.2/create-view.html#replace-an-existing-view) with the `CREATE OR REPLACE VIEW` syntax.  
+Core | **Foreign Key Performance and Compatibility** | When adding the [`FOREIGN KEY`](../v20.2/foriegn-key.html) constraint, it is no longer required to have an index on the referencing columns.
+Core | **`EXPLAIN` Improvements** | The response of the `EXPLAIN` statement now includes the estimated number of rows scanned by the query as well as other usability improvements.
+Core | **Disallowing Full Table Scans** | You can use the new [`disallow_full_table_scans` session variable](..v20.2/set-vars.html) to disallow full table and secondary index scans.   
+
+### Recovery and I/O
```

Comment:
Where do we typically communicate deprecations? The PM team is putting together this checklist due to lack of current best practice https://cockroachlabs.atlassian.net/wiki/spaces/PROD/pages/989200629/Feature+Deprecation+Checklist but at the very least we should do-so in the Release Notes.

In this case, `cockroach dump` is being deprecated since BACKUP is a much better backup options. For data export in "string" form (vs. backup is binary) they can use `EXPORT` and before we remove dump, we'll add functionality to print all schema at once (in the way that `dump` can today) as there's not direct replacement for that user story today.